TimeInput/README.md

```diff
@@ -0,0 +1,86 @@
+# TimeInput Interface
+
+**Status**: Proposed
+
+| Name         | ID                                                                |
+| :--------    | :---------------------------------------------------------------- |
+| TimeInput    | 047e749303e8a807c13a63c623567b028e822100805d9e6096aad71ada8ef27f  |
+
+
+## Description
+
+Allows to input day time.
+
+struct Time:
+
+    hrs: uint8 - number of the day. Must be in range [0, 24).
+
+    mins: uint8 - number of minutes. Must be in range [0, 60].
+
+    timestamp: uint32 - timestamp in seconds inside a 24 hours. Must be in range [0, 86400]. Used instead of hrs, mins.
+
+
+`get` - allows to get timestamp inside a day selected by user.
+
+arguments:
+
+	answerId: uint32 - function id of result callback
+
+    prompt: bytes - utf-8 string to print to the user before input.
+
+    minTime: Time - minimum time that can be chosen. NOTE: Time.timestamp can be zero if Time.hrs, Time.mins are defined.
+    
+    maxTime: Time - maximum time that can be chosen. NOTE: Time.timestamp can be zero if Time.hrs, Time.mins are defined.
+
+    interval: uint8 - minimal allowed interval between neibour times in minutes. Must be in range [1, 30].
+
+    timeZoneOffset: int16 - offset in minutes according to time zone.
+
+returns:
+
+	time: Time - chosen time.
+
+## Declaration in Solidity
+
+```jsx
+struct Time {
+    uint8 hrs;
+    uint8 mins;
```

Comment:
Since these two variables are actually just indexes of an hour and a minute within a day, I suggest to rename them as
```suggestion
    uint8 hour;
    uint8 minute;
```
since these are rather singular values than plural.